src/lib/api/base.js

```diff
@@ -29,6 +31,19 @@ const responseRejectInterceptor = error => {
     goTo(`${AuthenticationRoutes.login}?sessionExpired`);
   }
 
+  errorsList.forEach(errorCode => {
+    if (
+      error.response &&
+      error.response.status === errorCode &&
```

Comment:
Instead of the forEach, why not:
```
error.response && [404, 429, 500].includes(error.response.status)
```
see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes